### DISCUSSION TITLE
Macbook Air M1 chip, NFFT configure error: Unsupported mex file extension mexmaca64.

### DISCUSSION THREAD

**Discussion by Lu-Peter** [mtex-toolbox/mtex#2021] [Ask Anything] [answered]
I tried to configure NFFT with Matlab on Mac Air, M1 Chip. I tried many ways suggested by ChatGPT, but no success. Anyone can help out？ Thank you very much

./configure --enable-all --enable-openmp --with-fftw3=/opt/homebrew/Cellar/fftw/3.3.10_1 CC=/opt/homebrew/bin/aarch64-apple-darwin23-gcc-13 --with-sysroot=/Library/Developer/CommandLineTools/SDKs/MacOSX.sdk --with-libSystem_kernel=/usr/lib/system/libSystem_kernel.dylib --with-libSystem_platform=/usr/lib/system/libSystem_platform.dylib --with-libSystem_pthread=/usr/lib/system/libSystem_pthread.dylib --with-matlab=/Applications/MATLAB_R2023b.app

.......

checking for mexext... /Applications/MATLAB_R2023b.app/bin/mexext
checking for mex file extension... mexmaca64
checking for architecture... **configure: error: Unsupported mex file extension mexmaca64.**

kind regards
SL

**Comment by kilir** [comment]
Hi,
that's a good one. Since you decided to build arm binaries, I assume you have also Matlab running natively on arm, right?
Search the issues section, it's been covered multiple times and there are working binaries available.

Cheers,
Rüdiger

**Comment by Lu-Peter** [comment, marked as answer]
Hi Rüdiger
Thanks so much for your help. Problem solved after following suggestions here:
https://github.com/mtex-toolbox/mtex/discussions/1285#discussioncomment-5516403

Cheers
SL
